packages/vite-plugin-shopify/src/html.ts

```diff
@@ -114,25 +105,9 @@ export default function shopifyHTML (options: ResolvedVitePluginShopifyOptions):
 
 const viteTagDisclaimer = '{% comment %}\n  IMPORTANT: This snippet is automatically generated by vite-plugin-shopify.\n  Do not attempt to modify this file directly, as any changes will be overwritten by the next build.\n{% endcomment %}\n'
 
-// Generate liquid variable with resolved path by replacing aliases
-const viteTagEntryPath = (
-  resolveAlias: Array<{ find: string | RegExp, replacement: string }>,
-  entrypointsDir: string
-): string => {
-  const replacements: Array<[string, string]> = []
-
-  resolveAlias.forEach((alias) => {
-    if (typeof alias.find === 'string') {
-      replacements.push([alias.find, path.relative(entrypointsDir, alias.replacement)])
-    }
-  })
-
-  return `{% assign path = vite-tag | ${replacements.map(([from, to]) => `replace: '${from}/', '${to}/'`).join(' | ')} %}\n`
-}
-
 // Generate conditional statement for entry tag
-const viteEntryTag = (entryPaths: string[], tag: string, isFirstEntry = false): string =>
-  `{% ${!isFirstEntry ? 'els' : ''}if ${entryPaths.map((entryName) => `path == "${entryName}"`).join(' or ')} %}\n  ${tag}`
+const viteEntryTag = (entryName: string, tag: string, isFirstEntry = false): string =>
+  `{% ${!isFirstEntry ? 'els' : ''}if vite-tag == "${entryName}" %}\n  ${tag}`
```

Comment:
These changes were also necessary for the modules shorthand, i'd like to keep that feature